core/src/main/resources/lib/layout/layout.jelly

```diff
@@ -53,7 +53,7 @@ THE SOFTWARE.
       (The permission will be checked against the "it" object.)
     </st:attribute>
     <st:attribute name="type" use="optional">
-      Available values: two-column (by default) or one-column (full-width size).
+      Available values: two-column (by default), one-column (full-width size) or full-screen.
```

Comment:
Should mention which versions introduced these options, with the new one being `(since TODO)`